### PR TITLE
Fix a few lingering issues with doc conversion to new system

### LINKED ISSUE #12: Migrate BlackBear documentation to new system
Changes to BlackBear in support of migration to new MOOSE documentation system.

### PR DESCRIPTION
**Pull request by bwspenc**
closes #12

### CODE REVIEW

File: doc/content/documentation/systems/Materials/ConcreteDryingShrinkage.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 This material represents the eigenstrains induced by drying in concrete. The shrinkage strain $\epsilon_{sh}$ is calculated as:
-
-$$
+\begin{equation}
 \epsilon_{sh} = -k_{sh} (1 - h)^d
-$$
-
-Where $k_{sh}$ is the ``drying_shrinkage_coefficient`` (which must be positive), $h$ the relative humidity (a MOOSE variable which value must range between 0 and 1), and $d$ a positive ``exponent`` (typically equal to 1). The drying shrinkage is isotropic and reversible.
+\end{equation}
+where $k_{sh}$ is the `drying_shrinkage_coefficient` (which must be positive), $h$ the relative humidity (a MOOSE variable which value must range between 0 and 1), and $d$ a positive `exponent` (typically equal to 1). The drying shrinkage is isotropic and reversible.
```
[the remark this answers is not in the document]
Comment (bwspenc) [reply doc/content/documentation/systems/Materials/ConcreteDryingShrinkage.md]: Yes, they all are.

**Comment by bwspenc** [comment]
Pinging @sapitts -- does this all look good to you?

**Comment by sapitts** [comment]
Oh yep, it does.  I thought there was another reviewer assigned to this merge request

**Comment by sapitts** [comment]
Wait.  Shouldn't Civet have generated a docs page for this merge request?

**Comment by bwspenc** [comment]
Yes, the docs did get built.  Look for the link about 4 boxes up from your last comment.

**Comment by bwspenc** [comment]
Make that 3 boxes up!

**Comment by sapitts** [comment]
oh man, it's such a clear and well written comment that I completely missed the link!

**Comment by bwspenc** [comment]
Thanks!

**Pull request merged**